### ISSUE TITLE
NVDA not reading line properly when composing/replying to a post

### ISSUE THREAD

**Issue by johanntan** [trypsynth/fedra#6]
HI,
So when pressing NVDA L to read the current line, NVDA doesn't read the line fully in the composer.
Also, not sure if this is related or not, but the NVDA review cursor, not sure if that's how it's actually called or not, is not at the start of the line in the list of bosts, like when I press NVDA left or right its in the middle of the post instead of being at the top of the line.
